pdr_backend/models/feed.py

```diff
@@ -74,10 +74,10 @@ def dictToFeed(feed_dict: Dict[str, Any]):
         symbol=d["symbol"],
         seconds_per_epoch=int(d["seconds_per_epoch"]),
         seconds_per_subscription=int(d["seconds_per_subscription"]),
-        trueval_submit_timeout=int(d.get("trueval_submit_timeout", "100") or 100),
-        owner=d.get("owner", "") or "",
-        pair=d.get("pair", "") or "",
-        timeframe=d.get("timeframe") or "",
-        source=d.get("source") or "",
+        trueval_submit_timeout=int(d["trueval_submit_timeout"]),
```

Comment:
I'm fine with you reverting this, but it was an issue when running main.py (it's true, it doesn't cause problems with tests). We can reopen this if it occurs again, maybe it was something I published wrongly. If you want, we can create a ticket to track this, because without error handling it's possible that someone breaks the entire agent by publishing something unexpected/wrong.